package-lock.json

```diff
@@ -0,0 +1,1968 @@
+{
```

Comment:
It looks like you're using `npm` instead of `yarn` like @saphal1998 , and now you have both the `yarn.lock` and `package-lock.json` lockfiles in your project.

I would recommend standardising on one, because it can cause annoying churn in the project's dependencies when yarn and npm compete against each other in upgrading package versions.

My recommendation would be to use `yarn` since that's what `tsdx` uses, but I'm fine with either.

Reply:
I see, I'm fixing it to use yarn only!